src/main/java/net/datafaker/service/FakeValues.java

```diff
@@ -56,13 +69,26 @@ private static String language(Locale l) {
         return values == null ? null : (Map) values.get(key);
     }
 
+    private Map<String, Object> loadFromFilePath() {
+        if (filePath == null || !Files.exists(filePath) || Files.isDirectory(filePath) || !Files.isReadable(filePath)) {
+            return null;
```

Comment:
this method is called from `net.datafaker.service.FakeValues#loadValues` and it is a normal behavior for that method to return null there.
For user (in case a not valid filepath is passed there will be an `IllegalArgumentException` thrown from here https://github.com/datafaker-net/datafaker/pull/49/files#diff-6bc4a11e0bbd5b791bace06c0f9d0e7041b4a719956a773cab393d5b72cd963eR72)

Reply:
Sounds good, thanks for clarifying